config/theme.ts

```diff
@@ -6,6 +6,8 @@ export const theme_colors = {
     accent: '#7070702C',
     textLight: '#707070',
     tint: '#00E2C3',
+    tint2: 'rgba(0, 12, 235, 0.11)',
```

Comment:
Are we able to find a more descriptive name for this property instead of `tint2`?

Reply:
you could even call it something like `lightBlue`

Reply:
I used tint2 because on light mode it renders color blue and on dark mode it renders color white

Reply:
`secondaryTint`